outlines/processors/logging.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import sys
+import warnings
+from typing import List
+
+import torch
+
+from .base_logits_processor import OutlinesLogitsProcessor
+
+
+class LogitsLoggingLogitsProcessor(OutlinesLogitsProcessor):
+    """Handle chaining two logits processors to process logits sequentially"""
+
+    def __init__(self, tokenizer, top_n=8, logger=None, warn=True):
+        self.tokenizer = tokenizer
+        self.top_n = top_n
+        if logger is not None:
+            self.logger = logger
+        else:
+            self.logger = logging.getLogger("logits_logger")
+            self.logger.setLevel(logging.info)
```

Comment:
I believe this should be

```suggestion
            self.logger.setLevel(logging.INFO)
```